src/utils.h

```diff
@@ -145,7 +145,9 @@ extern SEXP (*rlang_env_dots_values)(SEXP);
 extern SEXP (*rlang_env_dots_list)(SEXP);
 
 void* r_vec_deref(SEXP x);
+void* r_vec_maybe_deref(SEXP x);
```

Comment:
Do we use it anywhere?

Reply:
I did at some point while I was experimenting. Removing for now.